queries/vim/iswap-list.scm

```diff
@@ -0,0 +1,4 @@
+[
+    (arguments)
```

Comment:
A lot of tree-sitter grammars define separate nodes for arguments and parameters. On my end, I can see a "parameters" node, for example, in the following function definition which has `a` and `b` as children:

```vim
function Test(a, b)
  echo 'hi'
endfunction
```

Update this PR if you can see the parameters node as well.

Reply:
I didn't think to check for that, I don't think I have any functions left in vimscript with multiple parameters. I've added them in now.